code/socialDistribution/templatetags/github_event.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+
+register = template.Library()
+
+# Django Software Foundation, "Custom Template tags and Filters", 2021-10-10
+# https://docs.djangoproject.com/en/3.2/howto/custom-template-tags/#inclusion-tags
+@register.inclusion_tag('tagtemplates/github_event.html')
+def card_gh_event(github_event):
```

Comment:
Do you think we could rename the HTML and the python function the same thing so that it's easier to find?